ui/app/AppLayouts/Communities/popups/BurnTokensPopup.qml

```diff
@@ -117,10 +135,81 @@ StatusDialog {
 
             ButtonGroup { id: radioGroup }
         }
+
+        StatusDialogDivider {
+            Layout.fillWidth: true
+        }
+
+        FeesBox {
+            id: feesBox
+
+            readonly property bool triggerFeeReevaluation: {
+                specificAmountButton.checked
+                amountToBurnInput.text
+                allTokensButton.checked
+                feesBox.accountsSelector.currentIndex
+
+                requestFeeDelayTimer.restart()
+                return true
+            }
+
+            Layout.fillWidth: true
+
+            placeholderText: qsTr("Choose number of tokens to burn to see gas fees")
+            accountErrorText: root.feeErrorText
+            implicitWidth: 0
+            model: d.isFormValid ? singleFeeModel : undefined
+            accountsSelector.model: SortFilterProxyModel {
```

Comment:
We have that sfpm for colors and filtering by type in many places. Not now but at some point we need to do some small refactor to have that proxy only in a single place, like CommunitySettingsView

Reply:
Yes, I totally agree. We have the `AccountSelector` component that would do the work.